tools/migration.py

```diff
@@ -0,0 +1,248 @@
+from __future__ import absolute_import
```

Comment:
please add a shebang to the file and make it executable so people can call it like ./migration.py directly
please add a comment at the beginning of this file, explaining what it does and how to use it